Source/Embeddings.Processing/EmbeddingProcessor.cs

```diff
@@ -174,7 +174,7 @@ async Task<Try> Loop()
             }
 
             var committedEvents = response.Events.ToCommittedEvents();
-            await replaceOrRemoveEmbedding(committedEvents.Last().AggregateRootVersion + 1).ConfigureAwait(false);
+            await replaceOrRemoveEmbedding(committedEvents.AggregateRootVersion).ConfigureAwait(false);
```

Comment:
`CommittedAggregateEvents.AggregateRootVersion` is the current aggregate root version, thus the same as the version of the last event + 1 (I know, we made it this confusing)